public/js/common.js

```diff
@@ -1,6 +1,13 @@
 function setUpNavbar() {
   // load navbar component
-  $('header').load('/components/navbar.html')
+  $('header').load('/components/navbar.html', function () {
+    var cartCount = localStorage.getItem('cart')
+    if (cartCount && $('#cartCount').length) {
+      $('#cartCount').addClass('badge').text(JSON.parse(cartCount).length)
+    } else {
+      $('#cartCount').removeClass('badge')
+    }
+  })
```

Comment:
Looks good dude!

Let's see if we can start to modularize the code now. Here's what I'm thinking:

Let's create a `services/cart-service.js` file.

This file has a `class CartService` and it also declares a global `cartService = new CartService()` variable that any page can access if it includes this js file as a `<script>`

That cart service can expose one method for now: `getCount()` which just returns an int.

In the future when we get to cart operations like adding/removing/querying we can add more methods to this class and include this script in other pages

Reply:
Added.